### ISSUE TITLE
Various signal errors in editor, related to TileMap [3.2.4-rc2]

### ISSUE THREAD

**Issue by snoopdouglas** [godotengine/godot#45938]
**Godot version:**
3.2.4.rc2.official

**OS/device including version:**
Ubuntu Linux 20.04

**Issue description:**
In the editor, when loading a scene that contains TileMaps - or adding a TileMap to an existing scene - 9 identical errors are written to the console:

```
 core/object.cpp:1257 - Error calling method from signal 'timeout': 'SceneTreeEditor::_update_tree': Method expected 1 arguments, but called with 0..
```

When deleting a TileMap, this error is printed twice:

```
 Attempt to disconnect a nonexistent connection to signal 'idle_frame' in [SceneTree:1228], with target '_on_workspace_process' in [TileSetEditor:15462].
```

**Steps to reproduce:**
As above.

**Minimal reproduction project:**
n/a

**Comment by bubutime** [comment]
same issue with popup node

**Comment by NeqsTech** [comment]
Can reproduce this bug up to this commit 1b9a01948e9f0470f4532c2de399a08b8b15f0b5. 
https://github.com/godotengine/godot/blob/1b9a01948e9f0470f4532c2de399a08b8b15f0b5/editor/scene_tree_editor.cpp#L1231

A `timeout` signal is connected to `SceneTreeEditor::_update_tree()` which now takes an optional parameter:

https://github.com/godotengine/godot/blob/1b9a01948e9f0470f4532c2de399a08b8b15f0b5/editor/scene_tree_editor.h#L77

It looks like the connection should look like this: 
`update_timer->connect("timeout", this, "_update_tree", varray(false)); `

**Comment by kleonc** [comment]
> When deleting a TileMap, this error is printed twice:
> 
> ```
>  Attempt to disconnect a nonexistent connection to signal 'idle_frame' in [SceneTree:1228], with target '_on_workspace_process' in [TileSetEditor:15462].
> ```
> 

This isn't fixed, still happens in the `3.3.stable`.

**Comment by akien-mga** [comment]
@kleonc Do you have more detailed steps to reproduce? Can't trigger this error when deleting a TileMap as described in the OP.

**Comment by kleonc** [comment]
@akien-mga Just creating and deleting TileMap is not enough (I haven't actually tested it earlier). As error points, it's a problem involving the TileSetEditor. Steps to reproduce:

1. Create new scene with TileMap as root.
2. Assign a TileSet to that TileMap and click it so the TileSetEditor will be shown at the bottom.
3. Delete that TileMap (don't left click / select TileMap before).

It seems that TileMap has nothing to do with it. Other way to trigger the error:
1. Create a TileSet resource.
2. Double click it to open TileSetEditor.
3. Create new scene.

This is where the error happens (line 3591):
https://github.com/godotengine/godot/blob/140cf0f2cb7b51d7866e63aba1aa6d8029cf540b/editor/plugins/tile_set_editor_plugin.cpp#L3583-L3593
Just checking if connection exist before connecting/disconnecting should fix this but maybe there's something wrong with the logic of the code calling `make_visible` in the first place, haven't investigate that.

**Comment by akien-mga** [comment]
Thanks, that issue doesn't appear to regression unlike the first one mentioned in the OP (it should have been filed as two separate issues).

>  Just checking if connection exist before connecting/disconnecting should fix this but maybe there's something wrong with the logic of the code calling `make_visible` in the first place, haven't investigate that.

Yeah I'm not sure why editing a TileSet resource doesn't trigger a `make_visible(true)` call that would connect the signal, but adding checks shouldn't hurt and at least solve this issue.

**Comment by akien-mga** [comment]
Fixed by #46031 and #48609.
